### ISSUE TITLE
Release 0.10.0

### ISSUE THREAD

**Issue by daedalus28** [FredrikNoren/ungit#650]
It looks like there's some really cool stuff in the pipe (like the branch picker, partial commits, etc) but there hasn't been a release in months.

Any ETA on when we can a 0.9.4 (or 0.10.0)?

**Comment by codingtwinky** [comment]
I'm all for it but knowing that there are a lot of new features coming in, if you would like to help us feel more comfortable releasing it via testing the latest it would be very helpful

Let me know if you have any questions on how you can help and I be help you to help you get the latest.

**Comment by FredrikNoren** [comment]
I think a new release would make sense. I created a milestone for everything that absolutely needs to be fixed for 0.10.0: https://github.com/FredrikNoren/ungit/milestones/0.10.0 I'll add issues as I find them, but as @codingtwinky says all the help people can give to find and fix more is appreciated.

Also: @codingtwinky that little spring animation for nodes is amazing :joy: I just spent five min checking out branches to see the effect.
